tests/Unit/Pdf/PdfTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace EMS\CommonBundle\Tests\Unit\Pdf;
+
+use EMS\CommonBundle\Service\Pdf\Pdf as PdfPdf;
+use PHPUnit\Framework\TestCase;
+
+class PdfTest extends TestCase
```

Comment:
The Pdf class is  also a "model" class.  We defined that we would not test Getters/Setters.